docs/source/notebooks/Boson_Bunching.ipynb

```diff
@@ -0,0 +1,1076 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Boson Bunching\n",
+    "\n",
+    "A prominent example of boson bunching is the Hong-Ou-Mandel effect, where the bunching of two photons arises from a destructive quantum interference between the trajectories where they both either cross a beam splitter or are reflected. This effect takes its roots in the indistinguishability of identical photons. You can read more about it [here](http://arxiv.org/abs/1711.00080) [1].\n",
```

Comment:
Instead of putting the link for the article each time, you can have a link from [1] to the bottom page (see docs/source/notebooks/Graph_States_generation_and_display.ipynb for how it's done)